src/main/scala/ai/privado/entrypoint/CommandParser.scala

```diff
@@ -51,10 +51,12 @@ object CommandConstants {
   val DISABLE_DEDUPLICATION_ABBR = "dd"
   val IGNORE_EXCLUDE_RULES       = "ignore-exclude-rules"
   val IGNORE_EXCLUDE_RULES_ABBR  = "ier"
+  val RETRY_SYNC                 = "retry-sync"
```

Comment:
@abhstabs The command name `retry-sync` doesn't relate directly to uploading the `privado.json` and metrics. 
we can give a thought on `privado upload` or `privado sync-result`, `privado upload-result`
cc - @pandurangpatil